app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DatasetsOverviewActivityTest.kt

```diff
@@ -53,15 +64,44 @@ class DatasetsOverviewActivityTest {
 
     @Test
     fun fragmentIsDisplayedWhenActivityIsLaunched() {
-        assertDisplayed(R.id.dataOverview_fragment)
-        assertDisplayed(R.id.dataOverviewButton)
+        assertDisplayed(R.id.datasetsOverview_fragment)
+        assertDisplayed(R.id.datasetsOverviewButton)
     }
 
     @Test
-    fun rightActivityIsStartedAfterCreateButton() {
-        onView(withId(R.id.dataOverviewButton)).perform(click())
+    fun rightActivityIsStartedAfterCreateButtonIsClicked() {
+        onView(withId(R.id.datasetsOverviewButton)).perform(click())
         intended(hasComponent(CategoriesEditingActivity::class.java.name))
+        onView(withId(R.id.button_submit_list)).perform(click())
+    }
+
+    @Ignore("TODO: enable when DatasetListWidget and DisplayDatasetActivity use the same DB")
+    @Test
+    fun whenClickOnDatasetDisplayDatasetActivity() {
+        waitFor(10000)
```

Comment:
Is such a big timeout really needed ? 
If it's due to the firestore db being slow, try injecting the dummy database, like in https://github.com/HumanLearning2021/HumanLearningApp/blob/9ea143a33d1f342f9e5c49a4c23de4cdf723c60f/app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningDatasetSelectionTest.kt#L33-L44